legacyCore/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,26 @@
+<resources>
+    <style name="Theme.Forzenbook" parent="Theme.Forzenbook.colorBase">
+        <!-- sizeing -->
+        <item name="smallPad_1">@dimen/paddingSmall2</item>
+        <item name="smallPad_2">@dimen/paddingSmall3</item>
```

Comment:
Attribute names should be camel case also for all the ones that aren't removed. Remember, snake case for the file name, camel for the name